ppa/archive/management/commands/generate_textcorpus.py

```diff
@@ -0,0 +1,262 @@
+"""
+**generate_textcorpus** is a custom manage command to generate a plain
```

Comment:
please add this to the sphinx docs - you can put it near where the other manage commands are listed

Reply:
should I remove `generate_corpus` or add `generate_textcorpus` beside it?

Reply:
Is that Vineet's script? My inclination is to remove it. Can you imagine any scenario where we'd want to run that now that we have the new version?